pyproject.toml

```diff
@@ -31,13 +31,13 @@ classifiers = [
 ]
 dynamic = ["version"]
 dependencies = [
-  "dvc>=3.33.3",
+  "dvc@git+https://github.com/iterative/dvc.git@refs/pull/10303/head",
   "dvc-render>=1.0.0,<2",
-  "dvc-studio-client>=0.17.1,<1",
```

Comment:
You probably didn't wanna push those changes (on `dvc` and `dvc-studio-client`) :)